master_component_list.json

```diff
@@ -5,1896 +5,2316 @@
         "mass": "132",
         "volume": "2700",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
     {
         "file": "Pacifier_Milspec_Package",
         "categoryname": "upgrades/Packages/Milspec",
         "price": "800000",
         "mass": "490",
         "volume": "10000",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
     {
         "file": "Progeny_Milspec_Package",
         "categoryname": "upgrades/Packages/Milspec",
         "price": "800000",
         "mass": "62",
         "volume": "1200",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
     {
         "file": "Schroedinger_Milspec_Package",
         "categoryname": "upgrades/Packages/Milspec",
         "price": "2000000",
         "mass": "80",
         "volume": "1600",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
     {
         "file": "Shizu_Milspec_Package",
         "categoryname": "upgrades/Packages/Milspec",
         "price": "200000",
         "mass": "57",
         "volume": "1140",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
     {
         "file": "Taizong_Milspec_Package",
         "categoryname": "upgrades/Packages/Milspec",
         "price": "3500000",
         "mass": "230",
         "volume": "4600",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
     {
         "file": "Zhuangzong_Milspec_Package",
         "categoryname": "upgrades/Packages/Milspec",
         "price": "1400000",
         "mass": "65",
         "volume": "1300",
-        "description": "Full-Custom Milspec Configuration"
+        "description": "Full-Custom Milspec Configuration",
+        "upgrade": true
     },
-    
-    
     {
         "file": "beam_heavy_lr",
         "categoryname": "upgrades/Weapons/Capship",
         "price": "2500000",
         "mass": "0.01",
         "volume": "1",
-        "description": "@upgrades/beam1.image@The logical conclusion of decades of upscaling work with spatial disruption technology, this weapon finds its home only among the weapons mounted on the capital vessels of well funded space navies."
+        "description": "@upgrades/beam1.image@The logical conclusion of decades of upscaling work with spatial disruption technology, this weapon finds its home only among the weapons mounted on the capital vessels of well funded space navies.",
+        "upgrade": true,
+        "weapoon": true,
+        "weapon": true
```

Comment:
@royfalk this is what I meant by `both` seems to be all throughout from here on down

Reply:
Not sure why this is not marked as obsolete, but when visually inspecting the changes, this is gone.